tests/manage/z_cluster/test_resource_request_and_limit_for_crashcollector_pod.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.resources.storage_cluster import StorageCluster
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from ocs_ci.helpers.helpers import (
+    verify_rook_ceph_crashcollector_pods_where_rook_ceph_pods_are_running,
+)
+from ocs_ci.framework.testlib import (
+    tier2,
+    ignore_leftovers,
```

Comment:
For my understanding, why do we need to ignore leftovers here? what will be left after the test?

Reply:
Made necessary changes